build/Dockerfile.nginx

```diff
@@ -0,0 +1,20 @@
+# syntax=docker/dockerfile:1.4
+FROM nginx:1.25.1-alpine
+
+ARG NJS_DIR
+ARG NGINX_CONF_DIR
+
+RUN apk add --no-cache libcap \
+    && mkdir -p /var/lib/nginx /usr/lib/nginx/modules \
+    && setcap 'cap_net_bind_service=+ep' /usr/sbin/nginx \
+    && setcap -v 'cap_net_bind_service=+ep' /usr/sbin/nginx \
+    && apk del libcap
+
+COPY ${NJS_DIR}/httpmatches.js /usr/lib/nginx/modules/njs/httpmatches.js
+COPY ${NGINX_CONF_DIR}/nginx.conf /etc/nginx/nginx.conf
+
+RUN chown -R 101:1001 /etc/nginx /var/cache/nginx /var/lib/nginx
+
+USER 101:1001
+
+STOPSIGNAL SIGTERM
```

Comment:
This should be the default stop signal https://docs.docker.com/engine/reference/commandline/stop/